src/parser/parser.yy

```diff
@@ -162,7 +162,7 @@ static constexpr size_t kCommentLengthLimit = 256;
 %token KW_BOOL KW_INT8 KW_INT16 KW_INT32 KW_INT64 KW_INT KW_FLOAT KW_DOUBLE
 %token KW_STRING KW_FIXED_STRING KW_TIMESTAMP KW_DATE KW_TIME KW_DATETIME
 %token KW_GO KW_AS KW_TO KW_USE KW_SET KW_FROM KW_WHERE KW_ALTER
-%token KW_MATCH KW_INSERT KW_VALUE KW_VALUES KW_YIELD KW_RETURN KW_CREATE KW_VERTEX KW_VERTICES
+%token KW_MATCH KW_INSERT KW_VALUE KW_VALUES KW_YIELD KW_RETURN KW_CREATE KW_VERTEX KW_VERTICES KW_IGNORE_EXISTED_INDEX
```

Comment:
emm.. This `IGNORE_EXISTED_INDEX` keyword is a little complicated to use, maybe it's better to split it into some words such as:

```
INSERT IGNORE EXISTS INDEX VERTEX ...
```

reference mysql insert ignore statement.

Reply:
I understand, but this is for almost the only one poc. They have used it since 1.X... Maybe just keep it.